_config.yml

```diff
@@ -8,10 +8,11 @@ title: "HackIllinois"
 description: > # this means to ignore newlines until "email:"
    The HackIllinois blog! Check out what we've been up to.
 
+future: true
+
```

Comment:
We're doing this so that the post goes live once we merge, and then we can publicize it on Facebook/Twitter once we're ready.